docs/features/sharepoint.md

```diff
@@ -0,0 +1,81 @@
+
+#### Document Support
+This ingestion process is only for **documents supported by our current pipeline**, the current officially supported file types are; pdf, docx, html, htm, csv, md, pptx, txt, json, xlsx, xml, eml, msg. This ingestion process does **NOT** include lists or information found on pages in your sharepoint site. It is only for documents.
+
+#### Files are Stored in Blob Storage
+Your sharepoint files will be stored in the blob storage created by Information Assistant, they are not going directly into AI Search. **There will be a copy of all your sharepoint files you are ingesting in blob storage.**
+
+### New Local Dev Variables
+
+#### Sharepoint Sites
+
+This is a comma delimited list of sharepoint site(s) you wish to ingest. **It shares a one-to-one relationship with the sharepoint folders variable.** If you have one sharepoint site, you need one sharepoint folder and vice versa. You may list the same site twice if you wish to more granular control over what is being crawled in sharepoint.
+
+#### Sharepoint Folders
+
+This is a comma delimited list of sharepoint folder(s) you wish to be your entry point for the ingestion. **It shares a one-to-one relationship with the sharepoint sites variable.** If you have one sharepoint folder, you need one sharepoint site and vice versa. It will crawl this folder and then automatically crawl any folders beneath it. The sharepoint site root is typically /Shared Documents
+
+#### Accepted File Types
+
+The sharepoint ingestion uses the same ingestion pipeline as Information Assistant to process the files obtained from sharepoint. The default values provided are those that have official support; pdf, docx, html, htm, csv, md, pptx, txt, json, xlsx, xml, eml, msg
+
+## Usage Instructions
+
+1. Fill in the sharepoint_sites, sharepoint_folders variables in your environment file. An example of that might look like this
+
+`export SHAREPOINT_SITES="https://wwpubsec.sharepoint.com/sites/SharepointTest, https://wwpubsec.sharepoint.com/sites/SharepointTest"`
+
+`export SHAREPOINT_FOLDERS="/Shared Documents/Example1, /Shared Documents/Example2"`
+
+Notice the 1:1 relationship between the sites and folders. Each site has **ONLY ONE** folder.
+
+2. Deploy Information Assistant like normal
+3. Once your deployment is complete, you'll need to login with a user that has access to the sharepoint sites you listed above. It is **strongly** recommended that you have created a new user for this purpose.
+4. Go to your resource group in the [Azure Portal](https://portal.azure.com/), select the "sharepointonline" API Connection resource.
+5. Click "Edit API Connection" in the menu on the left side of your screen.
+6. Click "Authorize" and login with the user that has access to the sharepoint sites you put in your environment file. It is **strongly** recommended that you have created a new user for this purpose.
```

Comment:
Is there no way to pass in a service principal instead of having to manually log in?

Reply:
Unfortunately not. Microsoft is retiring ACS (Access Control Services), which is how a service principal used to access Sharepoint. But that would've taken even more manual intervention to get setup. https://learn.microsoft.com/en-us/sharepoint/dev/solution-guidance/security-apponly-azureacs